### PR TITLE
Console command and Paper UI support for Thing disabled feature

### LINKED ISSUE #6440: Finish Thing disable feature
As per https://github.com/eclipse/smarthome/pull/5191#issuecomment-426230783, we have a new feature, which isn't really usable as it is lacking Paper UI and console integration - imho it actually should not have been merged without these features.

@YordanDZhelev May I ask you again whether you could add those features? If you are not fluent in the Paper UI code, I'd hope that @htreu could help on that.

### PR DESCRIPTION
**Pull request by htreu**
Fixes #6440.

### CODE REVIEW

File: bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/ThingManagerImpl.java [hunk 1]
```diff
@@ -634,6 +634,7 @@ public void run() {
     private void initializeHandler(Thing thing) {
         if (storage != null && storage.containsKey(thing.getUID().getAsString())
                 && !storage.get(thing.getUID().getAsString())) {
+            setThingStatus(thing, buildStatusInfo(ThingStatus.UNINITIALIZED, ThingStatusDetail.DISABLED));
```
Comment (htreu) [inline bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/ThingManagerImpl.java]: Set `DISABLED` state after restart

**Comment by maggu2810** [comment]
The code looks good to me but I didn't checked the Paper UI one, as I am not familiar with it.

**Comment by htreu** [comment]
For testing Paper UI on your terminal change to `extensions/ui/org.eclipse.smarthome.ui.paper` and run `npm run dev`. This will start Paper UI from a nodeJS web server on port 3000.

**Comment by maggu2810** [comment]
@htreu Thanks for the info. I already know how to run the latest Paper UI code, but I cannot judge if the code is "good" by reviewing it. I could only state "it seems to work" by testing it.

**Comment by htreu** [comment]
> I cannot judge if the code is "good" by reviewing it

Unfortunately the new code just follows the legacy style of this part of Paper UI. I did start a huge refactoring for the things details view this morning but could not reliably get it finished. And in the end it was too much of a change for this small feature. I will do the refactoring in a separate PR "in the next days(TM)".

**Pull request merged**